mod-linked-data/src/main/resources/karate-config.js

```diff
@@ -12,6 +12,7 @@ function fn() {
   var testTenant = karate.properties['testTenant'];
 
   var config = {
+    tenantParams: {loadReferenceData: true},
```

Comment:
Setting `loadReferenceData: true` will initialize all the reference data for the tenant when the tenant is initialized.

So, we do not have to load the authority files and instance types programmatically, as we did earlier.